ast/src/parsed/mod.rs

```diff
@@ -54,6 +55,7 @@ pub enum Expression<T> {
     Number(T),
     String(String),
     Tuple(Vec<Expression<T>>),
+    LambdaExpression(Vec<String>, Box<Expression<T>>),
```

Comment:
I think it's better to define a struct for LambdaExpression where we can document what each thing is

Reply:
We didn't do that for any of the stuff here

Reply:
We did for `FunctionCall`, and I think it's the clearest way to document what the arguments are here

Reply:
Also this should be renamed to `Lambda`